### PR TITLE
docs(core-deployment): translate deployment.md in English

### PR DESCRIPTION
**Pull request by gztchan**
##### Checklist

- [x] create deployment.md in core
- [x] translate deployment.md
- [x] refine translation

##### Affected core subsystem(s)

`docs/source/en/core/deployment.md`

##### Description of change

translation

### CODE REVIEW

**Comment by gztchan** [comment]
@fengmk2 @dead-horse @popomore Please review the translation of deployment.md.

File: docs/source/en/core/deployment.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+Both the application and dependencies will be packed into a tgz file, what you are going to do is unzipping and launching it.
+
+Reusable package brings a few pros in:
+- Environments in building and runtime are different, try to keep the later environment pure and stable.
+- Abbreviating publish progress and making rollback without hassle.
+
+## Deploy
+
+Node.js(>= 6.0.0) is required so that you should make sure it is pre-installed in runtime environment.
```
Comment (DarrenWong) [inline docs/source/en/core/deployment.md]: might need to follow the markdown syntax `>= 6.0.0`  ?

File: docs/source/en/core/deployment.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
+title: Deployment
+---
+
```
Comment (DarrenWong) [inline docs/source/en/core/deployment.md]: how about the translation between title and build?

Comment (gztchan) [inline docs/source/en/core/deployment.md]: That's my mistakes. I am going to finish it later. Thanks 👍

File: docs/source/en/core/deployment.md (hunk 1 above)
Comment (gztchan) [inline docs/source/en/core/deployment.md]: `>= 6.0.0` is better. 👍

**Comment by popomore** [comment]
Can you add translation about this PR? https://github.com/eggjs/egg/pull/1246

**Comment by gztchan** [comment]
#1246 has been translated. :0

**Comment by atian25** [comment]
@gztchan could you help for this update? https://github.com/eggjs/egg/pull/1279

**Comment by gztchan** [comment]
Sure 👍

**Pull request merged**